src/npctalk.cpp

```diff
@@ -2632,9 +2652,14 @@ void talk_effect_fun_t<T>::set_revert_location( const JsonObject &jo, const std:
             for( int y = 0; y < 2; y++ ) {
                 tripoint_abs_sm revert_sm = project_to<coords::sm>( omt_pos );
                 revert_sm += point( x, y );
-                const submap *sm = MAPBUFFER.lookup_submap( revert_sm );
-                get_timed_events().add( timed_event_type::REVERT_SUBMAP, tif, -1, revert_sm, 0, "",
-                                        sm->get_revert_submap() );
+                submap *sm = MAPBUFFER.lookup_submap( revert_sm );
+                if( sm == nullptr ) {
+                    get_map().load( revert_sm, true );
```

Comment:
Isn't `get_map()` the reality bubble itself? Why does this reload the reality bubble?

Reply:
Not quite.  This forces the submap to be loaded if it isn't already.  Encountered crashes otherwise.